scripts/build/daedalus-bridge.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
+echo "Building Daedalus Bridge..."
+
+# Cleaning node modules, bower components, etc.
+./scripts/clean/daedalus-bridge.sh
+
+# We have to build Cardano SL because there's a code in Haskell
+# codebase that needs to be recompiled for Bridge building.
+./scripts/build/cardano-sl.sh
```

Comment:
It's better to pass `sl` argument here for two reasons:
• it should be faster than `b` without arguments, because in this case packages will be built in parallel;
• new reason: after recent PR (which was merged this night) we also have `lwallet` which shouldn't be needed for daedalus.